misc/keyvaluestorage/package.json

```diff
@@ -40,18 +40,23 @@
     "lint": "eslint -c '../../.eslintrc' --fix './src/**/*.ts'"
   },
   "dependencies": {
-    "better-sqlite3": "^7.1.2",
-    "keyvaluestorage-interface": "^1.0.0",
     "localStorage": "^1.0.4",
     "safe-json-utils": "^1.1.1"
   },
+  "optionalDependencies": {
+    "@react-native-async-storage/async-storage": "1.x",
+    "better-sqlite3": "7.x"
+  },
   "devDependencies": {
-    "@walletconnect/time": "^1.0.1",
+    "@react-native-async-storage/async-storage": "^1.17.3",
```

Comment:
I think async storage and better-sqlite3 need to become `peerDependencies` with flexible major versions like `^1`

Reply:
I've done this above with `optionalDependencies`, although `peerDependencies` with maybe [peerDependenciesMeta](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#peerdependenciesmeta) is actually want we want here.

The common pattern is `peerDeps + devDeps` or `optionalDeps + devDeps`, since we need these deps to build and run tests during development.

Reply:
I think it might be `optionalDependencies` because depending on the platform you only need one of those right?

Reply:
Yeah tbh it's somewhat confusing what the actual difference between peerDeps + peerDependenciesMeta set to `optional` vs simply `optionalDependencies` is